[Docs migration] Cleans up investigation tools files - pt 2 (#565)
elastic/docs-content: solutions/security/investigate/cases-requirements.md

  - https://www.elastic.co/guide/en/serverless/current/security-cases-requirements.html
---

# Cases requirements [security-cases-requirements]

::::{note}
- To send cases to external systems, ensure you have the appropriate [{{stack}} subscription](https://www.elastic.co/pricing) or [{{serverless-short}} project tier](../../../deploy-manage/deploy/elastic-cloud/project-settings.md).

> - Certain subscriptions and privileges might be required to manage case attachments. For example in {{stack}}, to add alerts to cases, you must have privileges for [managing alerts](/solutions/security/detect-and-alert/detections-requirements.md#enable-detections-ui). In {{serverless-short}}, you need the Security Analytics Complete [project feature](../../../deploy-manage/deploy/elastic-cloud/project-settings.md).

Author: ```suggestion
- Certain subscriptions and privileges are required to manage case attachments. For example in {{stack}}, to add alerts to cases, you must have privileges for [managing alerts](/solutions/security/detect-and-alert/detections-requirements.md#enable-detections-ui). In {{serverless-short}}, you need the Security Analytics Complete [project feature](../../../deploy-manage/deploy/elastic-cloud/project-settings.md).
```
